etc/db_files/tables.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE IF NOT EXISTS tags(
+	id INTEGER PRIMARY KEY,
+	tag VARCHAR(32) UNIQUE
+);
+
+CREATE TABLE IF NOT EXISTS pomodoro(
+	-- id INTEGER PRIMARY KEY,
+	tag_id INTEGER REFERENCES tags(id),
+	dt_start DATETIME NOT NULL,
+	dt_end DATETIME NOT NULL,
+	descript VARCHAR(512)
+);
+
+CREATE TABLE IF NOT EXISTS statistic(
+	-- id INTEGER PRIMARY KEY,
+	label VARCHAR(32),
+	dt_start DATETIME NOT NULL,
+	dt_end DATETIME NOT NULL
+);
```

Comment:
I don't think we want to have this file here at this stage. We will need to discuss the data structure later... for now, you can move it to `tests/samples/db_files` and use it for test only.